apps/src/aiTutor/views/AITutorFloatingActionButton.tsx

```diff
@@ -39,7 +39,7 @@ const AITutorFloatingActionButton: React.FunctionComponent = () => {
         type="button"
       />
       <Provider store={store}>
-        <AITutorPanel open={isOpen} />
+        <AITutorContainer open={isOpen} closeTutor={handleClick} />
```

Comment:
Passing this handler enables a close with an "X" in the upper right-hand corner.